content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -100,6 +104,42 @@ On your validator:
 
 
 
+## Connect to the Network Using Public Hubs
+
+This section applies to running a validator that you want to remain an untrusted validator. In this case, it may be more efficient for you to use public hubs to connect your validator to the network than to run stock `rippled` servers to use as proxies.
+
+**Note:** If you are running a trusted validator or a validator that you want to become trusted, [connect your validator to the network using proxies](#connect-to-the-network-using-proxies) instead of public hubs.
+
+As a validator operator, one of your key responsibilities is to ensure that your validator has reliable and safe connections to the XRP Ledger network. Instead of connecting to random and potentially malicious peers on the network, you can instruct your validator to connect to the network through one or more public hubs. This setup can help protect your validator from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks.
+
+A public hub is a `rippled` server that Ripple has assessed to have a track record of embodying the following traits: ***TODO: accurate to say that Ripple assessed this?***
```

Comment:
I would position this as 'traits of public hubs' and leave it a little vague. Probably would avoid mentioning Ripple assessed the hubs.